contracts/evoting/types/admin.go

```diff
@@ -62,24 +57,27 @@ func (adminForm *AdminForm) AddAdmin(userID string) error {
 }
 
 // GetAdminIndex return the index of admin if userID is one, else return -1
-func (adminForm *AdminForm) GetAdminIndex(userID string) int {
-	sciperInt, err := strconv.Atoi(userID)
+func (adminForm *AdminForm) GetAdminIndex(userID string) (int, error) {
+	sciperInt, err := SciperToInt(userID)
 	if err != nil {
-		return -1
+		return -1, xerrors.Errorf("Failed SciperToInt: %v", err)
 	}
 
 	for i := 0; i < len(adminForm.AdminList); i++ {
 		if adminForm.AdminList[i] == sciperInt {
-			return i
+			return i, nil
 		}
 	}
 
-	return -1
+	return -1, nil
 }
 
 // RemoveAdmin add a new admin to the system.
 func (adminForm *AdminForm) RemoveAdmin(userID string) error {
-	index := adminForm.GetAdminIndex(userID)
+	index, err := adminForm.GetAdminIndex(userID)
+	if err != nil {
+		return xerrors.Errorf("Failed GetAdnubIndex: %v", err)
```

Comment:
please change to a more comprehensible error message

Reply:
oupsi for the typo, it's updated 👍